src/index.html

```diff
@@ -3,11 +3,478 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Eco-cosmetics</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="shortcut icon" href="./images/company-logo.svg">
   </head>
   <body>
-    <h1>Hello Mate Academy</h1>
+
+    <header class="header">
+
+      <div class="header__top">
+        <span class="logo">
+          <img
+            class="logo__img"
+            src="./images/company-logo.svg"
+            alt="logo eco-cosmetics"
+          >
+          <span
+            class="logo__text"
+          >
+            Witch of mountain winds
+          </span>
+        </span>
+
+        <button
+          type="button"
+          class="header__menu-button"
+          aria-label="menu-button"
+          aria-expanded="false"
+          aria-controls="first-mobile"
+          data-menu-button="menu"
+        >
+          <div class="header__hamburger"></div>
+        </button>
+
+        <nav class="nav header__nav" data-menu="menu">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#shop"
+              >
+                Shop
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#contact"
+              >
+                Contact
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#about"
+              >
+                About
+              </a>
+            </li>
+          </ul>
+        </nav>
+
+        <span>
+          <a href="#" class="shopping link">
+            <img
+              class="header__shopping-bag"
+              src="./images/shopping-bag.svg"
+              alt="logo shopping-bag"
+            >
+          </a>
+        </span>
+      </div>
+
+    </header>
+
+    <main class="main">
+
+      <section class="section">
+        <div class="brand">
+          <div class="brand__left">
+            <h1
+              class="brand__heading"
+            >
```

Comment:
Якщо один чи два атрибути, то можеш не переносити
```suggestion
            <h1 class="brand__heading">
```